downloads/kr/github-git-cheat-sheet.md

```diff
@@ -54,45 +54,45 @@ http://git-scm.com
 
 {% capture colTwo %}
 
-## 변경하기
-수정 사항을 리뷰하고 커밋 기록을 생성합니다
+## 변경점을 저장하기
+수정 사항을 검토하고 커밋을 생성합니다
 
 
 ```$ git status```
 
-커밋 대상의 모든 새로운 파일 혹은 수정된 파일 리스트를 보여줍니다
+커밋할 수 있는 새로운 파일과 수정된 파일의 목록을 보여줍니다
 
 ```$ git diff```
 
-아직 Staged 되지 않은 파일의 차이점을 보여줍니다
+수정하였으나 아직 stage하지 않은 파일의 변경점을 보여줍니다
 
 
 ```$ git add [file]```
 
-버저닝(versioning) 준비를 위해 파일을 일시적으로 저장(snapshot)합니다
+커밋을 준비하기 위해 파일을 stage합니다
 
 
 ```$ git diff --staged```
 
-곧 커밋할 것이라고 표시(staging)한 것과 가장 최근 파일 버전 간의 차이점을 보여줍니다
+stage하였으나 아직 커밋하지 않은 파일과 가장 최근에 커밋한 파일을 비교합니다
 
 
 ```$ git reset [file]```
 
-파일을 언스테이지(unstage) 상태로 변경시키나 파일 컨텐츠는 그대로 보존합니다
+파일의 내용은 유지한 채로 stage한 내역만을 제거합니다
 
 
 ```$ git commit -m"[descriptive message]"```
 
-버전 히스토리에 파일 스냅샷을 영원히 기록합니다
+stage한 내용을 커밋으로 영구히 저장합니다
 
-## 그룹 변경하기
-커밋 시리즈의 이름을 설정하고 결과물을 결합시킵니다
+## 변경점을 묶어 관리하기
+일련의 커밋에 이름을 붙이고 여러 변경점을 합칩니다
 
 
 ```$ git branch```
 
-현재 저장소 안의 모든 로컬 브랜치 리스트를 보여줍니다
+현재 저장소의 모든 로컬 브랜치를 보여줍니다
 
 ```$ git branch [branch-name]```
 
```

Comment:
새로운 브랜치를 생성합니다.

Reply:
Yeah, the lack of periods in the previous (Korean) document has irked me, but the original (English) doc doesn't have them either. Since it is a cheat sheet after all, I guess punctuation marks aren't as necessary.